src/overlays/actors/ovl_Obj_Aqua/z_obj_aqua.c

```diff
@@ -66,7 +66,7 @@ Vec3f D_80ACC308 = { 1.0f / 1000.0f, 7.0f / 10000.0f, 1.0f / 1000.0f };
 Vec3f D_80ACC314 = { 8.6f / 1000.0f, 8.0f / 10000.0f, 8.6f / 1000.0f };
 Vec3f D_80ACC320 = { 1.0f / 100.0f, 2.6f / 1000.0f, 1.0f / 100.0f };
 
-extern Gfx D_0407D590[];
+extern Gfx gGameplayKeepDrawFlameDL[];
```

Comment:
I know it wasn't you, but this should be removed and the gameplay_keep header should be used instead